### PR TITLE
Add changelog and basic documentation for SSAI tracking

### PR DESCRIPTION
**Pull request by strangesource**
## Problem
SSAI ad tracking is not yet documented

## Changes
- Add basic readme entry for SSAI ad tracking
- Add changelog entry for SSAI ad tracking

## Related Changes
- #70 
- #72 
- #73 
- #74 
- #75
- #76

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+### Added
+- `ConvivaAnalyticsIntegration.ssai` namespace to enable server side ad tracking
```
Comment (rolandkakonyi) [inline CHANGELOG.md:9]: Did you intentionally go for a minimal entry here?
For iOS I listed every API with a very short description.

I chose a more verbose approach to the changelog entry but realized just now that I missed to update the readme which in your case compliments this, so this is more of a general question.

File: README.md [hunk 2]
```diff
@@ -133,6 +133,37 @@ A session can be ended using following method call:
...
+```
+
+In addition to the metadata provided in the `AdInfo` object at ad start, the following metadata will be auto collected from the main content metadata:
+- ConvivaSdkConstants.STREAM_URL
+- ConvivaSdkConstants.ASSET_NAME
+- ConvivaSdkConstants.IS_LIVE
+- ConvivaSdkConstants.DEFAULT_RESOURCE 
+- ConvivaSdkConstants.ENCODED_FRAMERATE
+- streamType
+- integrationVersion
```
Comment (rolandkakonyi) [inline README.md:163]: Should we mention the out-of-the-box auto-collected values as well for the viewerId and player name?

File: CHANGELOG.md (hunk 1 above)
Comment (strangesource) [reply CHANGELOG.md:9]: I felt like the functions names are quite self explanatory, hence I did not put them here. But no strong opinion, happy to align with you changelog for consistency.

File: README.md (hunk 2 above)
Comment (strangesource) [reply README.md:163]: Good call. 👍

File: CHANGELOG.md (hunk 1 above)
Comment (rolandkakonyi) [reply CHANGELOG.md:9]: My goal was to showcase, so if you only look at changelog it doesn't look much with a one-liner but also no strong opinion.

Comment (rolandkakonyi) [reply CHANGELOG.md:9]: Let's go with this simple one-liner as it feels heavy with the README together when having both verbose!

**Review by rolandkakonyi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -133,6 +133,37 @@ A session can be ended using following method call:
 `convivaAnalyticsIntegration.endSession()`
 Since the `BitmovinPlayer` automatically pauses the video if no background playback is configured the session creation after the app is in foreground again is handled automatically.
 
+#### Server Side Ad Tracking
```
Comment (rolandkakonyi) [inline README.md:136]: nit: Why is `####` is used under `##` right away?

Comment (strangesource) [reply README.md:136]: Good question, I just aligned with surrounding headlines.

Comment (strangesource) [reply README.md]: [386d858](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/76/commits/386d858b80bedb774e519f4057a9ca4843eb177c)

File: README.md (hunk 2 above)
Comment (strangesource) [reply README.md:165]: [2a447b0](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/76/commits/2a447b0258698ed630e42224c6fcd51ebf19fcb2)

**Pull request merged**